CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Check podfile sources and plugin sources for warning.
```

Comment:
nit: please add two empty spaces after the period for markdown formatting.

Reply:
done